docs/workforce-management-framework/topics/con-performance-guidelines.adoc

```diff
@@ -0,0 +1,85 @@
+[id='{context}-performance-guidelines']
+= Performance Guidelines
+
+{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that aren't part of the standard applications.
+
+Node.js based server can rely on customer prefered database or external api and any other datasource. This  may have significant impact on overal application performance. {PROJECT_NAME} was desigined to expose areas and configurations that can have .
```

Comment:
Typo for 'preferred', capitalize 'API', double spacing.

Also seems like the sentence is incomplete here.